ONSdigital/eq-survey-runner
# Build Census Household Accommodation

### What is the context of this PR?
[trello](https://trello.com/c/5V7lUw1C/3184-introduce-hh-schema-build-accommodation-eng-wls-m)

This PR adds a household schema and adjusts the build process for it.

Currently the build_schema script will error because the NI household schema does not exist. I'm not sure what the best solution is here. I didn't want to copy all of the blocks / schema over.

I have included a built version in `data/en` this will be deleted before merging, it is only there to help with quick-launch.

### How to review 
- Check the jsonnet all makes sense
- Check the build script makes sense
- Check the rendered schema matches the prototype / changes in trello.
- Make sure routing and validation work as specified.

Quick-launch: https://rich-launcher.gcp.dev.eq.ons.digital/quick-launch?language_code=en&survey=CENSUS&case_type=HH&region_code=GB-ENG&url=https://raw.githubusercontent.com/ONSdigital/eq-survey-runner/3184-build-accommodation/data/en/census_household_gb_eng.json


### Checklist
